docs/site/Appsody-LoopBack.md

```diff
@@ -0,0 +1,990 @@
+---
+lang: en
+title: 'Deploying to Cloud with Appsody'
+keywords: LoopBack 4.0, LoopBack 4, Appsody
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Appsody-LoopBack.html
+---
+
+## LoopBack 4 & Appsody
+
+[Appsody](https://appsody.dev/docs#welcome-to-appsody) is designed to help
+develop containerized applications for the cloud.
+
+The [Appsody CLI](https://appsody.dev/docs/using-appsody/cli-commands/) makes it
+extremely easy for a developer to: scaffold, run, stop, debug, test, build, and
+deploy an application to a cloud.
+
+The
+[Node.js LoopBack stack](https://github.com/appsody/stacks/tree/master/incubator/nodejs-loopback)
+extends the
+[Node.js stack](https://github.com/appsody/stacks/tree/master/incubator/nodejs)
+and provides a powerful solution to build open APIs and Microservices in
+TypeScript with [LoopBack](https://loopback.io/), an open source Node.js API
+framework. It is based on
+[LoopBack 4](https://github.com/strongloop/loopback-next).
+
+We're going to show you how to scaffold, run, stop, debug, and test a LoopBack 4
+application locally. Then we're going to show you how to build and deploy this
+application to the [IBM Cloud](https://www.ibm.com/cloud).
```

Comment:
Seeing `We're going to show you` twice. Can we use a different phrase?

Reply:
`We're going to show you how to scaffold, run, stop, debug, and test a LoopBack 4
application locally. Then, you'll learn how to build and deploy this
application to the [IBM Cloud](https://www.ibm.com/cloud).`